packages/CollabCodeEdit-Server.package/CCEServer.class/class/serverPort.st

```diff
@@ -1,3 +1,5 @@
 constants
 serverPort
-	^ (ThisOSProcess thisOSProcess environmentAt: 'CCE_SERVER_PORT') ifNil: [ 8080 ]
\ No newline at end of file
+	^ (ThisOSProcess thisOSProcess environmentAt: 'CCE_SERVER_PORT')
```

Comment:
Cant we use a normal class/instance variable here?

Reply:
No, this is configuration that has to be passed in from the environment.